python/lib/dependabot/python/update_checker/pipenv_version_resolver.rb

```diff
@@ -88,7 +85,7 @@ def fetch_latest_resolvable_version_string(requirement:)
                 # Whilst calling `lock` avoids doing an install as part of the
                 # pipenv flow, an install is still done by pip-tools in order
                 # to resolve the dependencies. That means this is slow.
-                run_pipenv_command("pyenv exec pipenv lock")
+                run_pipenv_command("pyenv exec pipenv lock --dev")
```

Comment:
FWIW if we wanted to make this work for python 2, we'd need to pass in `--two` as an option to all the `pipenv` commands when `python_version.start_with?("2")`

Reply:
👍  would really like to avoid keeping v2 support. Would also block these updates https://github.com/dependabot/dependabot-core/pull/3775 and https://github.com/dependabot/dependabot-core/pull/3498

Reply:
pipenv will probably follow suite and drop v2 support soon https://github.com/pypa/pipenv/issues/4261#issuecomment-768866753